salt/scripts.py

```diff
@@ -81,14 +81,14 @@ def suicide_when_without_parent(parent_pid):
     try:
         minion = salt.cli.daemons.Minion()
         minion.start()
+    except (ImportError, SystemExit) as exc:
+        log.error(exc)
```

Comment:
Ah, I didn't read up enough. I do like it being explicit but you're right. Thanks!